package.json

```diff
@@ -81,7 +82,9 @@
     "react-qr-svg": "^2.1.0",
     "react-string-replace": "^0.4.4",
     "rimraf": "^2.7.1",
-    "simple-markdown": "^0.7.0",
+    "run-parallel": "^1.1.9",
+    "run-series": "^1.1.8",
```

Comment:
@Jikstra didn't I removed them on purpose?

Reply:
Probably i resolved the merge conflicts wrongly